modules/yarn.lock

```diff
@@ -10265,6 +10265,11 @@ lie@~3.3.0:
   dependencies:
     immediate "~3.0.5"
 
+liferay-ckeditor@4.18.0-liferay.4:
+  version "4.18.0-liferay.4"
+  resolved "https://registry.yarnpkg.com/liferay-ckeditor/-/liferay-ckeditor-4.18.0-liferay.4.tgz#08ddc7c1025f6c036a9de97c3403b9928e6b99dc"
+  integrity sha512-mugcikGLtVbZ1kqxtHjf1z8F4hWPeET4EyCx48ePIUc01fnOuAEiiNwnaWeBxiMS8K75CvYNbA37jJhCskHXmw==
+
```

Comment:
Why did this get added?

Reply:
no idea, something (running `yarn` in `modules` maybe) added it automatically 🤷

Reply:
I will `yarn why` it

Reply:
@bryceosterhaus `yarn why liferay-ckeditor` inside modules:
```
=> Found "liferay-ckeditor@4.18.0-liferay.4"
info Reasons this module exists
   - "_project_#@liferay#frontend-js-dependencies-web" depends on it
   - Hoisted from "_project_#@liferay#frontend-js-dependencies-web#liferay-ckeditor"
```

Reply:
ah I see. maybe just didn't get added before 

PR looks fine to me, just need CI or QA sign off

Reply:
We recently [updated CKEditor](https://github.com/liferay-frontend/liferay-portal/pull/3701) to 4.21.0-liferay.1. It looks like we forgot to update all dependencies cc @fortunatomaldonado 

@kresimir-coko can you please update it in `frontend-js-dependencies-web`

Reply:
sure!

Reply:
https://github.com/liferay-frontend/liferay-portal/pull/3724/commits/7714187f5e9aa8cc93483fa3e54dbb0c7046550a good enough? @markocikos

Reply:
Perfect! :sparkles: